File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/cot/Java/Python/s545439898_comments_removed.py
import sys

def main():
    count = int(input())
    giving_numbers = [int(input()) for _ in range(count)]

    for i in range(count - 1, -1, -1):
        print(giving_numbers[i], end='')
        if i > 0:
            print(' ', end='')
    print()

if __name__ == "__main__":
    main()
# End of Code

File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/direct/Java/Go/s545439898_comments_removed.go
package main

import (
	"fmt"
)

func main() {
	var count int
	fmt.Scan(&count)
	givingNumbers := make([]int, count)

	for i := 0; i < count; i++ {
		fmt.Scan(&givingNumbers[i])
	}

	for i := count - 1; i >= 0; i-- {
		fmt.Print(givingNumbers[i])
		if i > 0 {
			fmt.Print(" ")
		}
	}

	fmt.Println()
}
// End of Code
